aptos-move/framework/aptos-framework/sources/Account.move

```diff
@@ -34,6 +35,8 @@ module AptosFramework::Account {
         currency_code_required: bool,
     }
 
+    struct SignerCapability has drop, store { account: address }
```

Comment:
Is SignerCap droppable so people can just drop the cap as a way to remove ownership in their script/module code? An alternative here is not to make it droppable and provide a function to destroy SignerCap in Account. I actually prefer the latter more as it makes it very explicit that someone is destroying the SignerCap.

Reply:
Hmm... Lemme follow up on this. It is late and I want to make the cut.